database/rrdhost.c

```diff
@@ -712,10 +712,16 @@ static int is_valid_label_key(char *key) {
     if(!strcmp(key, "chart") || !strcmp(key, "family")  || !strcmp(key, "dimension"))
         return 0;
 
-    //Internal dimension
+    //Netdata internal
     if (*key == '_')
         return 0;
 
+    key++;
+    //Prometheus internal
+    if (*key == '_')
+        return 0;
+
+    key++;
```

Comment:
i think there is no reason to check for `__` if `_` is already wrong (return 0). And i think you dont need `key++` after the check.

This is perfect case to use unit tests btw

Reply:
I removed with the last commit, you right, I had a test that did not reach the loop when there was one unique `_` , so the second is not necessary.
I agree this is a perfect place for unique test.